### PR TITLE
Adjust to app.mayhem.security.

### PR DESCRIPTION
**Pull request by eddymul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,7 +167,7 @@ The mCode Action provides the following outputs:
 | `runId` | string | The identifier of the run that this action triggered in Mayhem. |
 
 
-📖 See the [CI/CD](https://mayhem.forallsecure.com/docs/mayhem-ci-cd/fuzzing-in-your-pipeline/) docs for more information and guides on using the mCode GitHub Action!
```
[the remark this answers is not in the document]
Comment (eddymul) [reply README.md:168]: Should be resolved in [e889175](https://github.com/ForAllSecure/mcode-action/pull/116/commits/e889175d04104de5b87477de2e59b018d2191f84)

But, would appreciate another look. 🙏

File: README.md [hunk 2]
```diff
@@ -12,20 +12,22 @@ A GitHub Action for using Mayhem for Code to check for reliability, performance
...
 ## Getting Started
 
 To use the Mayhem for Code GitHub Action, perform the following steps:
 
-1. Navigate to [mayhem.forallsecure.com](https://mayhem.forallsecure.com/) to register an account.
+1. Navigate to [app.mayhem.security](https://app.mayhem.security/) to register an account.
 
-    1. Click your profile drop-down and go to *Settings* > *API Tokens* to access your account API token.
+    1. After logging in, on the bottom left:
+       1. click your profile icon.
```
Comment (Andrew5194) [inline README.md:24]:
```suggestion
       1. Click your profile icon.
```

File: README.md [hunk 3]
```diff
@@ -12,20 +12,22 @@ A GitHub Action for using Mayhem for Code to check for reliability, performance
...
+       2. click Account Settings.
```
Comment (Andrew5194) [inline README.md:25]:
```suggestion
       2. Click Account Settings.
```

**Review by Andrew5194 — APPROVED** [review]
LGTM, left a few suggestions

**Pull request merged**